src/figuresClasses.ts

```diff
@@ -2,10 +1,110 @@
+function roundTo(n: number, decimals: number): number {
+  const factor = Math.pow(10, decimals);
 
-export class Triangle implements Figure {}
+  return Math.floor(n * factor) / factor;
+}
+
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  color: 'red' | 'green' | 'blue';
+
+  a: number;
+
+  b: number;
+
+  c: number;
+
+  constructor(
+    color: 'red' | 'green' | 'blue',
+    a: number,
+    b: number,
+    c: number,
+  ) {
+    this.shape = 'triangle';
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides must be positive numbers');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('This cannot be a valid triangle');
+    }
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+
+    return Math.sqrt(s * (s - this.a) * (s - this.b) * (s - this.c));
```

Comment:
The area calculation for the triangle should be rounded to the nearest hundredth. Use the `roundTo` function to achieve this.